monkestation/code/datums/station_traits/neutral_traits.dm

```diff
@@ -12,3 +12,20 @@
 /datum/station_trait/announcement_duke/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/duke
+
+/datum/station_trait/new_space_law
+	name = "New Space Law"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 10
+	show_in_report = TRUE
+	report_message = "A recent bet at centcom has led to a new space law being created. A report on it will be printed at all security record consoles."
+
+/datum/station_trait/new_space_law/on_round_start()
+	var/new_space_law = pick(world.file2list("monkestation/strings/new_space_laws.txt"))
+	priority_announce("Due to recent events in space politics [new_space_law] now 1xx illegal under space law.")
+	for(var/obj/machinery/computer/secure_data/console in GLOB.machines)
+		if(!(console.machine_stat & (BROKEN|NOPOWER)) && is_station_level(console.z))
+			var/obj/item/paper/lawprint = new /obj/item/paper(console.loc)
+			lawprint.name = "paper - 'New Space Law'"
+			lawprint.info = "Due to recent events in space politics [new_space_law] now 1xx illegal under space law."
```

Comment:
Here as well. Poor grammar and contradicting text.

Reply:
same as above